sop_security.wiki

```diff
@@ -44,18 +44,54 @@ For information about deadly force, trials, etc, see [[Space Law]].
+
+5. The Head of Security may not overrule a Magistrate, unless their decisions are blatantly breaking Standard Operating Procedure and/or Space Law. In which case Central Command is to be contacted as well;
+
+6. The Head of Security must follow the same guidelines as the Warden for Armory equipment, portable flashers and deployable barriers;
+
+7. The Head of Security is not permitted to collect equipment from the Armory to carry on their person;
+
+8. The Head of Security is permitted to either use their regular coat, or armored trenchcoat;
 
-2. The Head of Security is permitted to wear their Safeguard MODSuit at all times.
+9. The Head of Security is permitted to wear their unique gas mask;
+
+10. The Head of Security may not overrule established sentences, unless further evidence is brought to light or the prisoner in question attempts to escape;
+
+11. The Head of Security must follow the same guidelines as Security Officers for attire, with the exception that their own assigned attire is also acceptable for any of the attire mentioned for Security Officers.
+
+12. The Head of Security is permitted to wear their Safeguard MODSuit at all times.
 
 === <span style="color: darkred">Code Red</span> ===
 ----
 
-1. Guidelines 1, 4, 5, 6, 8, 9, 10, and 11 are carried over from Code Green;
+1. The Head of Security is permitted to carry out arrests under the same conditions as their Security Officers;
+
+2. The Head of Security is not obligated to provide a trial, but is encouraged to allow legal representation should the suspect request it. This only applies to Capital Crimes;
 
-2. Guideline 2 is carried over from Code Blue;
+3. The Head of Security may not overrule a Magistrate, unless their decisions are blatantly breaking Standard Operating Procedure and/or Space Law. In which case Central Command is to be contacted as well;
 
-3. The Head of Security is required to make a Station Announcement regarding the nature of the confirmed threat that caused Code Red. The Captain may perform this duty in the place of the Head of Security.
+4. The Head of Security must follow the same guidelines as the Warden for Armory equipment, portable flashers and deployable barriers;
+
+5. The Head of Security is permitted to either use their regular coat, or armored trenchcoat;
+
+6. The Head of Security is permitted to wear their unique gas mask;
+
+7. The Head of Security may not overrule established sentences, unless further evidence is brought to light or the prisoner in question attempts to escape;
+
+8. The Head of Security must follow the same guidelines as Security Officers for attire, with the exception that their own assigned attire is also acceptable for any of the attire mentioned for Security Officers.
+
+9. The Head of Security is permitted to wear their Safeguard MODSuit at all times.
+
+10. Guideline 2 is carried over from Code Blue;
```

Comment:
Was there a reason to keep that as a carry over or was that a mistake?

Reply:
yeah thats a mistake